air_mouse/air_mouse_app.c

```diff
@@ -1,129 +1,240 @@
 #include <furi.h>
 #include <furi_hal.h>
+#include <furi_hal_bt_hid.h>
+#include <bt/bt_service/bt.h>
 #include <gui/gui.h>
-#include <dialogs/dialogs.h>
+#include <gui/view_dispatcher.h>
+#include <gui/modules/submenu.h>
+#include <gui/modules/dialog_ex.h>
 #include "imu_mouse.h"
-#include "air_mouse_icons.h"
+#include "views/air_mouse_view.h"
+#include <furi_hal_usb_hid.h>
+#include <storage/storage.h>
 
 #define TAG "SensorModule"
 
+#define HID_BT_KEYS_STORAGE_NAME ".bt_hid.keys"
```

Comment:
There is a problem: this ble profile also used in hid remote with different credentials, so they are going conflict.

Reply:
`bt_keys_storage_set_storage_path(bt, APP_DATA_PATH(HID_BT_KEYS_STORAGE_NAME));`
This file goes to app data dir, which is different in hid remote app